lib/src/Datepicker/formatters.ts

```diff
@@ -0,0 +1,15 @@
+export function padNumber(value: number, isYear: boolean = false): string {
+    if (isNumber(value)) {
+        return isYear ? `0000${value}`.substr(-4, 4) : `00${value}`.substr(-2, 2);
+    } else {
+        return "";
+    }
+}
```

Comment:
Use this syntax, much cleaner
```javascript
(num).toLocaleString('en', { minimumIntegerDigits: 2, useGrouping: false })
```

Reply:
Nice suggestion, thanks! I thought it is not well supported by older browsers but it looks like it is.